templates/perfalert_inactive_regression_nag.html

```diff
@@ -0,0 +1,23 @@
+<p>
+    The following {{ plural('bug is', data, pword='bugs are') }} are performance alerts which have had no activity in the last 7 days:
```

Comment:
We could avoid hardcoding the number here as well. We could pass the number as extra similar to the following: https://github.com/mozilla/bugbot/blob/e12b1e13f2eca587c513fd02fb8e8538c33dcc59/bugbot/rules/leave_open_no_activity.py#L26-L28